core/src/main/java/com/scalar/db/config/DatabaseConfig.java

```diff
@@ -62,7 +60,6 @@
   public static final String PASSWORD = PREFIX + "password";
   public static final String STORAGE = PREFIX + "storage";
   public static final String TRANSACTION_MANAGER = PREFIX + "transaction_manager";
-  public static final String ISOLATION_LEVEL = PREFIX + "isolation_level";
```

Comment:
If we use `DatabaseConfig.ISOLATION_LEVEL`, we need to change it to `ConsensusCommitConfig.ISOLATION_LEVEL`. It's backward incompatibility but I think it's okay.